course_content/matplotlib_intro.ipynb

```diff
@@ -151,12 +151,12 @@
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "Notice how the axes' view limits (``ax.viewLim``) have been updated to include the whole of the line.\n",
-      "Should we want to add some spacing around the edges of our axes we could set the axes' margin using the Axes artist's [``margins``](http://matplotlib.org/api/axes_api.html?highlight=axes#matplotlib.axes.Axes.margins) method. Alternatively, we could manually set the limits with the Axes artist's [``set_xlim``](http://matplotlib.org/api/axes_api.html?#matplotlib.axes.Axes.set_xlim) and [``set_ylim``](http://matplotlib.org/api/axes_api.html?#matplotlib.axes.Axes.set_ylim) methods.\n",
+      "Notice how the axes view limits (``ax.viewLim``) have been updated to include the whole of the line.\n",
```

Comment:
Gramatically, is ```axes'``` not the correct form? The view limits belong to the ``axes``...